Makefile

```diff
@@ -1,24 +1,29 @@
-buf := ./bin/buf
 node_modules := $(shell npm root)
+buf := npm exec --offline -- buf
```

Comment:
Interesting. This command is supposed to mean...

> Run `buf` from `node_modules` and error out if it's not available

The little `$(node_modules)` dependency may be a little too clever. An alternative would be to remove the dependency and rely on users to run `make setup` manually when needed, rather than try to do it automatically based on if `node_modules` is present `package-lock.json` hasn't changed

Reply:
i think we still need the dependency to ensure that these commands run in CI, since both require `buf` to be installed through npm

Reply:
I'm okay with the current setup - if it causes problems for local dev we can revisit.